packages/sync-host/src/index.ts

```diff
@@ -53,19 +53,25 @@ export class HostSync extends Sync {
 
         this.iframe = iframe;
 
-        this.iframe.src = url + this.getState();
+        try {
+            const {origin, pathname, search} = new URL(url);
+            const updatedPathname = pathname === '/' ? '' : pathname;
+            this.iframe.src = `${origin}${updatedPathname}${this.getState()}${search}`;
 
-        !this.iframe['iFrameResizer'] &&
-            iFrameResize(
-                {
-                    checkOrigin: false,
-                    minHeight,
-                    heightCalculationMethod: 'min',
-                    sizeWidth: true,
-                    tolerance: 50,
-                },
-                this.iframe,
-            );
+            !this.iframe['iFrameResizer'] &&
+                iFrameResize(
+                    {
+                        checkOrigin: false,
+                        minHeight,
+                        heightCalculationMethod: 'min',
+                        sizeWidth: true,
+                        tolerance: 50,
+                    },
+                    this.iframe,
+                );
+        } catch (e) {
+            //ignore
```

Comment:
Not good. Why do we need to capture and omit? Lack of resizer means that some init did not happen, even that has to do `console.error`, btw, can you fix it too? If URL can't be constructed it should result in a meaningful error.

Reply:
Fixed. Please review again. I also remove the default value of min-height.

Reply:
@kirill-konshin, I found that this is the only place to call `iFrameResize()`.  I suppose we should not throw error here. Or else it will always throw the error.
Please correct me if I am wrong.